bigchaindb/config_utils.py

```diff
@@ -97,8 +98,11 @@ def file_config(filename=None):
         filename = CONFIG_DEFAULT_PATH
 
     logger.debug('file_config() will try to open `{}`'.format(filename))
-    with open(filename) as f:
-        config = json.load(f)
+    try:
+        with open(filename) as f:
+            config = json.load(f)
+    except Exception as err:
```

Comment:
Exception handling here is too broad. Usually a `try ... except Exception` is an [antipattern](https://realpython.com/blog/python/the-most-diabolical-python-antipattern/) to avoid. Also, the `FileNotFoundError` is used in `autoconfigure` at L215.

My suggestion here is to catch only the exceptions you expect.